cmd/repo_remove.go

```diff
@@ -36,7 +36,7 @@ You cannot remove the default repository, but you can make a different repositor
 				return errors.New("Error, you must specify repository name")
```

Comment:
Can't this be "You must specify the repository name." as the error text should be generated?